modules/ytp-assets-common/src/less/ckan/upstream_ckan/layout.less

```diff
@@ -65,6 +65,8 @@
     position: relative;
     float: right;
     margin-left: 0; // Remove grid margin.
+    padding-left:34px;
+    padding-right:6px;
```

Comment:
Less files under `upstream_ckan/` shouldn't be touched, so these should be moved into some other place